### PR TITLE
Fixed queue deletion through UI.

### PR DESCRIPTION
**Pull request by hemikak**
Fixed issue of queue not getting deleted from UI when deleted.
JIRA - https://wso2.org/jira/browse/MB-945

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java [hunk 1]
```diff
@@ -191,20 +207,51 @@ public void removeBinding(AndesBinding andesBinding) throws AndesException {
...
 
-    private void notifyExchangeListeners(AndesExchange exchange, ExchangeListener.ExchangeChange change) throws AndesException {
-        for(ExchangeListener listener : exchangeListeners) {
+    /**
+     * Notifying the exchange listeners stating that a change has occurred for the exchanges in the
+     * local node. This will then get notified throughout the cluster if clustered deployment is
+     * available.
+     *
+     * @param exchange The andes exchange in which the change occurred.
+     * @param change   The change that is being occurred
```
Comment (pamod) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java]: How about describing the possible types of exchanges here, (i.e Added,Deleted)

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java [hunk 2]
```diff
@@ -191,20 +207,51 @@ public void removeBinding(AndesBinding andesBinding) throws AndesException {
...
 
-    private void notifyBindingListeners(AndesBinding binding, BindingListener.BindingEvent change) throws AndesException {
-        for(BindingListener listener : bindingListeners) {
+    /**
+     * Notifying the bindings listeners stating that a change has occurred for the bindings in the
+     * local node. This will then get notified throughout the cluster if clustered deployment is
+     * available.
+     *
+     * @param binding The binding in which the change occurred.
+     * @param change  The change that occurred.
```
Comment (pamod) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java]: The again would be more descriptive to define the possible types of bindings here

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java (hunk 1 above)
Comment (hemikak) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java]: Will add the supported exchange types.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java (hunk 2 above)
Comment (hemikak) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java]: Yes, will add description comment on bindings.

Comment (hemikak) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java]: Fixed in https://github.com/wso2/andes/pull/58

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java (hunk 1 above)
Comment (hemikak) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java]: Fixed in https://github.com/wso2/andes/pull/58

**Pull request merged**